### ISSUE TITLE
Fira Code Retina being install as Medium

### ISSUE THREAD

**Issue by dublUayaychtee** [tonsky/FiraCode#1241]
I am using Garuda Linux, based off of manjaro I think, and when I install all the fonts, fira code retina shows up as Medium.
![image](https://user-images.githubusercontent.com/62506798/118727775-d95bc100-b822-11eb-8697-92e25ceed90b.png)

**Comment by tonsky** [comment]
Yes, Retina is a non-standard name and some systems have problem with it. Maybe report to them?
